pkg/blockatlas/tx.go

```diff
@@ -91,6 +93,8 @@ type Tx struct {
 	Meta interface{} `json:"metadata"`
 }
 
+// TxOutput describes input and output of tx
+// Excample: bitcoin UTXO
```

Comment:
`// Excample: bitcoin UTXO` why adding this? + misspelled

Reply:
ok removed